.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,93 @@
+name: Pre-commit actions for Delphix Virtualization Platform SDK
+
+on: [pull_request]
+
+jobs:
+  pytest:
+    name: Test ${{ matrix.package }} on ${{ matrix.os }} using pytest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [2.7]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
I think we should run this on windows too

Reply:
When i include windows, i get test failures in our existing tests in tools project. So, i think we can take up fixing them as separate task.

Reply:
What issue do you get on Windows?

Reply:
We talked offline. It's the actual unit tests that are failing. An issue will be filed to track that work separately.